### DISCUSSION TITLE
Deploying Keycloak with ISPN Cache on Kubernetes for High Availability

### DISCUSSION THREAD

**Discussion by ido808** [keycloak/keycloak#35832] [Q&A]
I’m deploying Keycloak (v26) on Kubernetes with 2 pods for High Availability and need guidance on configuring Infinispan (ISPN) caching for proper synchronization. I’ve searched for guides but couldn’t find any specific to Kubernetes so I'm currently running with 1 POD.

Could anyone share a step-by-step guide or best practices for setting this up? Any pointers to existing resources or templates would be greatly appreciated.

**Comment by embesozzi** [comment]
See the following guides:
- https://www.keycloak.org/high-availability/introduction
- https://www.keycloak.org/keycloak-benchmark/
- https://github.com/keycloak/keycloak-benchmark
